cmd/networkcmd/network.go

```diff
@@ -1,13 +1,16 @@
 // Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
-package cmd
+package networkcmd
 
 import (
 	"fmt"
 
+	this "github.com/ava-labs/avalanche-cli/pkg/app"
```

Comment:
I am not sure of using a package renamed to this for the app package. Probably will be best to decide on different names for the package and for the variable (not both app)

Reply:
I renamed to `application`